### PR TITLE
New and improved Readme

### LINKED ISSUE #144: Create a better README.md
Right now we're still using the stock `create-next-app` readme, and we should probably update that to have info about HaaS in general, setting up a basic dev environment, etc.

### PR DESCRIPTION
**Pull request by anddddrew**
Closes #144

### CODE REVIEW

**Comment by anddddrew** [comment]
Does it need anymore info?

**Review by cjdenio — CHANGES_REQUESTED** [review]
This looks great to start; could you just remove the dashes in "Hack as a Service"?

**Comment by anddddrew** [comment]
> This looks great to start; could you just remove the dashes in "Hack as a Service"?

Done!

File: README.md [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+    ```
+2. Change directory.
+   ``` 
+   cd frontend
+   ```
+3. Install dependencies. 
+   ```
+   yarn install
+   ```
+
```
Comment (anirudhb) [inline README.md:34]: Please make the note that the API server has to be setup (which can be done so by following its readme)

**Review by anirudhb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anddddrew) [reply README.md:34]: I will link the API repo then

File: README.md [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
+   ```
+   
+4. Start the server.
+   ```
+   yarn dev
+   ```
+Once you have completed all these steps, you should be good to go 🚀.
+You can make your desired changes, and create a pull request once you are done.
+
+[contributors-shield]: https://img.shields.io/github/contributors/othneildrew/Best-README-Template.svg?style=for-the-badge
```
Comment (anirudhb) [inline README.md:42]: Meant to change this?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
+[contributors-url]: https://github.com/hack-as-a-service/frontend/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/hack-as-a-service/frontend.svg?style=for-the-badge
+[forks-url]: https://github.com/hack-as-a-service//network/members
```
Comment (anirudhb) [inline README.md:45]: Extra slash

File: README.md [hunk 4]
```diff
@@ -1,34 +1,49 @@
...
+## Contributing
+
+How can I contribute?
+
+1. Clone the repo.
+   ```sh
+   git clone https://github.com/hack-as-a-service/frontend
+    ```
+2. Change directory.
+   ```
```
Comment (anirudhb) [inline README.md:27]: Small nit: do you mind adding `sh` syntax to these codeblocks as well?

**Comment by anddddrew** [comment]
@anirudhb Fixed, readme should be ready to go

**Review by anirudhb — APPROVED** [review]

**Review by cjdenio — APPROVED** [review]

**Pull request merged**